src/IssueViz.Security/Store/HotspotsStore.cs

```diff
@@ -45,10 +50,19 @@ internal class HotspotsStore : IHotspotsStore, IIssueLocationStore
     {
         private ObservableCollection<IAnalysisIssueVisualization> Hotspots { get; } = new ObservableCollection<IAnalysisIssueVisualization>();
 
-        void IHotspotsStore.Add(IAnalysisIssueVisualization hotspot)
+        IAnalysisIssueVisualization IHotspotsStore.GetOrAdd(IAnalysisIssueVisualization hotspot)
```

Comment:
The parameter name `hotspot` is a bit misleading since it's a visualization that we expect to contain a hotspot.
I think the visualization params normally have "Viz" in the name.